isis/src/base/objs/TrackingTable/TrackingTable.cpp

```diff
@@ -0,0 +1,165 @@
+/**
+ * @file
+ * $Revision: 1.4 $
+ * $Date: 2010/05/14 19:17:09 $
+ *
+ *   Unless noted otherwise, the portions of Isis written by the USGS are public
+ *   domain. See individual third-party library and package descriptions for
+ *   intellectual property information,user agreements, and related information.
+ *
+ *   Although Isis has been used by the USGS, no warranty, expressed or implied,
+ *   is made by the USGS as to the accuracy and functioning of such software
+ *   and related material nor shall the fact of distribution constitute any such
+ *   warranty, and no responsibility is assumed by the USGS in connection
+ *   therewith.
+ *
+ *   For additional information, launch
+ *   $ISISROOT/doc//documents/Disclaimers/Disclaimers.html in a browser or see
+ *   the Privacy &amp; Disclaimers page on the Isis website,
+ *   http://isis.astrogeology.usgs.gov, and the USGS privacy and disclaimers on
+ *   http://www.usgs.gov/privacy.html.
+ */
+
+#include "TrackingTable.h"
+
+#include <QList>
+#include <QString>
+
+#include "FileName.h"
+#include "IException.h"
+#include "Pvl.h"
+#include "Table.h"
+#include "TableField.h"
+#include "TableRecord.h"
+
+using namespace std;
+namespace Isis {
+
+
+  /**
+  * @brief Default constructor
```

Comment:
It doesn't matter, but the @brief tag is not really needed. Doxygen will pull the first sentence for the brief documentation when @brief isn't used.